src/main/java/speakql/SpeakQLSpeechlet.java

```diff
@@ -43,375 +34,410 @@
+   * events.
+   */
+  private static final String SESSION_TEXT = "text";
+
+  /**
+   * Constant defining session attribute key for the intent slot key for the
+   * date of events.
+   */
+  private static final String SLOT_DAY = "day";
+
+  /**
+   * Size of events from Wikipedia response.
+   */
+  private static final int SIZE_OF_EVENTS = 10;
+
+  /**
+   * Array of month names.
+   */
+  private static final String[] MONTH_NAMES = {
+      "January",
+      "February",
+      "March",
+      "April",
+      "May",
+      "June",
+      "July",
+      "August",
+      "September",
+      "October",
+      "November",
+      "December"
+  };
+
+  @Override
+  public void onSessionStarted(
+      final SessionStartedRequest request, final Session session)
+      throws SpeechletException {
+    log.info("onSessionStarted requestId={}, sessionId={}",
+        request.getRequestId(), session.getSessionId());
+  }
+
+  @Override
+  public SpeechletResponse onLaunch(
+      final LaunchRequest request, final Session session)
+      throws SpeechletException {
+    log.info("onLaunch requestId={}, sessionId={}", request.getRequestId(),
+        session.getSessionId());
+
+    return getWelcomeResponse();
+  }
+
+  @Override
+  public SpeechletResponse onIntent(
+      final IntentRequest request, final Session session)
+      throws SpeechletException {
+    log.info("onIntent requestId={}, sessionId={}", request.getRequestId(),
+        session.getSessionId());
+
+    Intent intent = request.getIntent();
+    String intentName = intent.getName();
+
+    if ("TableAggregateIntent".equals(intentName)) {
+      return TableAggregateIntent(intent, session);
+    } else if ("GetNextEventIntent".equals(intentName)) {
+      return handleNextEventRequest(session);
+    } else if ("HelpIntent".equals(intentName)) {
+      // Create the plain text output.
+      String speechOutput =
+          "With SpeakQL, you can query"
+              + " your database for aggreagtes, group bys, and order bys"
+              + " For example, you could say,"
+              + " How many nights booked did we have in the past week.";
+
+      String repromptText = "Which day do you want?";
+
+      return newAskResponse(
+          "<speak>" + speechOutput + "</speak>",
+          "<speak>" + repromptText + "</speak>");
+    } else if ("FinishIntent".equals(intentName)) {
+      SsmlOutputSpeech outputSpeech = new SsmlOutputSpeech();
+      outputSpeech.setSsml("<speak> Goodbye </speak>");
+      return SpeechletResponse.newTellResponse(outputSpeech);
+    } else {
+      throw new SpeechletException("Invalid Intent");
     }
-
-    @Override
-    public SpeechletResponse onLaunch(final LaunchRequest request, final Session session)
-            throws SpeechletException {
-        log.info("onLaunch requestId={}, sessionId={}", request.getRequestId(),
-                session.getSessionId());
-
-        return getWelcomeResponse();
```

Comment:
its still there i just changed the beginning of this file to include variables that were deleted for some reason and were throwing errors because of it